### ISSUE TITLE
Implement triggering action after pressing combination of controller buttons

### ISSUE THREAD

**Issue by pktiuk** [AntiMicroX/antimicrox#298]
### Discussed in https://github.com/AntiMicroX/antimicrox/discussions/297

<div type='discussions-op-text'>

<sup>Originally posted by **milochadam** November 21, 2021</sup>
I would like to use a trigger as a modifier, for example holding it down would allow faster mouse movement or right trigger + left stick up would result in increasing volume (while left stick without right trigger moves mouse cursor around)</div>

**Comment by pktiuk** [comment]
There are many features AmtiMicroX is missing, but my time is limited.  
Currently, I am focused on testing and delivering motion sensor support.

Feel free to contribute and open your own PR :)

**Comment by pktiuk** [comment]
Before you start messing with code I recommend checking our short guideline https://github.com/AntiMicroX/antimicrox/wiki/Coding-guidelines

**Comment by rexendevar** [comment]
I agree, this would easily be the game-changer for me - I already have similar behavior set up using one button to move to a set while holding and another to trigger the action from this new set - but this makes the number of sets a limiting factor as well as the number of buttons I am willing to assign to separate sets. It also means that one of the shortcuts I have, turning off AMX behavior entirely by moving to an empty set, takes two separate nearly-blank sets to implement and requires me to press entirely different buttons to turn it off vs to turn it back on. It is absolutely workable but kind of a pain to attempt to control lots of things without it.

**Comment by DanielSpaniel8** [comment]
You can set the right trigger to "select set 2 while held" in the advanced options, then in set 2 increase the mouse movement speed. I do this sort of thing a lot, and it works pretty well.

**Comment by qmzmdqs** [comment]
I also need this functionality. This is the key to simply operating the computer instead of the keyboard

**Comment by eleriaqueen** [comment]
Oh dang this feature isn't implemented yet. I am trying to set a Playstation Navigation Controller up because I have pain in my right hand when using a mouse and as the PSNC has a joystick I want to map L-Button + Stick Up/Down to scrollwheel.
